### PR TITLE
Add metrics to record the latency of Deer workers

### PR DESCRIPTION
**Pull request by Dalamar42**
- These measure the time between the moment the message was generated
  and the time it finished processing

### CODE REVIEW

File: atlas-processing/src/main/java/org/atlasapi/messaging/ContentEquivalenceUpdatingWorker.java [hunk 1]
```diff
@@ -47,6 +50,7 @@ private ContentEquivalenceUpdatingWorker(
         this.executionTimer = metricRegistry.timer(metricPrefix + "timer.execution");
         this.messageReceivedMeter = metricRegistry.meter(metricPrefix + "meter.received");
         this.failureMeter = metricRegistry.meter(metricPrefix + "meter.failure");
+        this.latencyTimer = metricRegistry.timer(metricPrefix + "latency.meter");
```
Comment (jamiesprax) [inline atlas-processing/src/main/java/org/atlasapi/messaging/ContentEquivalenceUpdatingWorker.java]: Follow convention and use "meter.latency"? Unless you are doing it this way for other reasons

File: atlas-processing/src/main/java/org/atlasapi/messaging/EquivalentContentIndexingContentWorker.java [hunk 2]
```diff
@@ -51,6 +52,7 @@ private EquivalentContentIndexingContentWorker(
         this.executionTimer = metricRegistry.timer(metricPrefix + "timer.execution");
         this.messageReceivedMeter = metricRegistry.meter(metricPrefix + "meter.received");
         this.failureMeter = metricRegistry.meter(metricPrefix + "meter.failure");
+        this.latencyTimer = metricRegistry.timer(metricPrefix + "latency.meter");
```
Comment (jamiesprax) [inline atlas-processing/src/main/java/org/atlasapi/messaging/EquivalentContentIndexingContentWorker.java]: +naming here

File: atlas-processing/src/main/java/org/atlasapi/messaging/EquivalentContentIndexingGraphWorker.java [hunk 3]
```diff
@@ -43,6 +45,7 @@ private EquivalentContentIndexingGraphWorker(
         this.executionTimer = metricRegistry.timer(metricPrefix + "timer.execution");
         this.messageReceivedMeter = metricRegistry.meter(metricPrefix + "meter.received");
         this.failureMeter = metricRegistry.meter(metricPrefix + "meter.failure");
+        this.latencyTimer = metricRegistry.timer(metricPrefix + "latency.meter");
```
Comment (jamiesprax) [inline atlas-processing/src/main/java/org/atlasapi/messaging/EquivalentContentIndexingGraphWorker.java]: +naming here

File: atlas-processing/src/main/java/org/atlasapi/messaging/ContentEquivalenceUpdatingWorker.java (hunk 1 above)
Comment (Dalamar42) [inline atlas-processing/src/main/java/org/atlasapi/messaging/ContentEquivalenceUpdatingWorker.java]: Gagh... Good catch. Will fix

File: atlas-processing/src/main/java/org/atlasapi/messaging/EquivalentContentIndexingGraphWorker.java (hunk 3 above)
Comment (jamiesprax) [inline atlas-processing/src/main/java/org/atlasapi/messaging/EquivalentContentIndexingGraphWorker.java]: I'll stop pointing it out now...

**Pull request merged**